WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -200,12 +200,67 @@ private extension SupportTableViewController {
         return ImmuTable(sections: sections)
     }
 
+    // TODO - remove after FeatureFlag.wordPressSupportForum is removed
+    func oldTableViewModel() -> ImmuTable {
+
+        // Help Section
+        var helpSectionRows = [ImmuTableRow]()
+        helpSectionRows.append(HelpRow(title: LocalizedText.wpHelpCenter, action: helpCenterSelected(), accessibilityIdentifier: "help-center-link-button"))
```

Comment:
I wonder if we should be keeping this "Help Center", there's a lot of helpful information there about the functionality 🤔. What are your thoughts?

Reply:
Thanks for starting the discussion. I think we can stick with the designs and what is implemented in this PR. Reintroduce this Help & Center row if needed. 👍

Reply:
No problem 🙇